Localization/Translations/DebugMessages.lua

```diff
@@ -234,7 +234,7 @@ local debugMessagesLocales = {
         ["ptBR"] = "Atualização do banco de dados do Questie concluída!",
         ["ruRU"] = "Обновление базы данных Questie завершено!",
         ["deDE"] = "Questie Datenbank-Update abgeschlossen!",
-        ["koKR"] = nil,
+        ["koKR"] = "Questie DB 업데이트 완료!!!",
```

Comment:
Are multiple `!` a thing in Korean? If not just make it a single one 😄 
```suggestion
        ["koKR"] = "Questie DB 업데이트 완료!",
```